mastahnish/BlackMagicShineDemo
# Feature/exoplayer integration

**Acceptance criteria**
Finished First Part of the task. Implemented ExoPlayer presenting the video from url buffering it over http.

**How to test**
Run the app and see the below video:
https://user-images.githubusercontent.com/11060323/162856342-fee77ec4-051b-4c90-8007-72f94850650c.mp4

**Next steps**
1. Implement initial 4second animation showing countdown
2. Implement Location Services and combine them with MediaPlayer (reset video functionality)
3. Implement gyroscope ability and combine it with MediaPlayer controls (volume in axis x and seek in axis z)
4. Implement sensor service to listen to shake event and based on that stop the video
